cql3/expr/expression.cc

```diff
@@ -1468,5 +1468,13 @@ utils::chunked_vector<std::vector<managed_bytes_opt>> get_list_of_tuples_element
 
     return tuples_list;
 }
+
+expression to_expression(const ::shared_ptr<term>& term_ptr) {
+    if (term_ptr.get() == nullptr) {
+        return constant::make_null();
```

Comment:
Why is a nullptr term equivalent to a CQL NULL? Isn't it an internal error?

Reply:
`nullptr` `terminal` means `null`.
For example in `constants::null_literal::null_value::bind`:
https://github.com/scylladb/scylla/blob/acf8da2bcef47168376e876e2fdc37fc26ebc82e/cql3/constants.hh#L86

Later it can be cast to term and then `to_expression` could be called on it.
In that case a nullptr `term` means a valid null value.

Reply:
Ok. I'll be happy when we can get rid of term.